src/utils/sykmeldinger/sykmeldingUtils.ts

```diff
@@ -161,11 +161,13 @@ export const arbeidsgivernavnEllerArbeidssituasjon = (
   }
 };
 
-export const sykmeldingerMedStatusSendt = (
+export const getSendteOrNyeSykmeldinger = (
```

Comment:
Tenkte det ga mer mening at en funksjon var mer verb enn substantiv, men gjerne kom opp med bedre navneforslag!

Reply:
Jeg bare fjernet den, ble bare brukt her og da trenger man ikke en egen util + util-test for dette. Testes heller rett i komponenten.